tmcprototype/CspMasterLeafNode/CspMasterLeafNode.xmi

```diff
@@ -28,7 +28,7 @@
       <status abstract="false" inherited="true" concrete="true"/>
       <DefaultPropValue>localhost</DefaultPropValue>
     </deviceProperties>
-    <deviceProperties name="CspMasterFQDN" description="FQDN of the CSP Master TANGO Device Server. Value of this property &#x2192; tango://mid-csp-tango:10000/mid_csp/elt/master">
+    <deviceProperties name="CspMasterFQDN" description="FQDN of the CSP Master TANGO Device Server. Value of this property &#x2192; tango://mid_csp-tango:10000/mid_csp/elt/master">
```

Comment:
Are we required to give hostname since we are using only one TANGO facility? 
It may cause issues in K8s deployment, I am not sure.

Reply:
Agree. There is no need to write TANGO facility. I will remove it.